usaspending_api/references/management/commands/load_guide.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import os.path
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from openpyxl import load_workbook
+
+from usaspending_api.references.models import Definition, DefinitionResource
+
+
+class Command(BaseCommand):
+    help = "Loads an Excel spreadsheet of USAspending terminology definitions into the Guide model"
+
+    logger = logging.getLogger('console')
+
+    def add_arguments(self, parser):
+        path = 'usaspending_api/data/USAspendingGuide.xlsx'
+        path = os.path.normpath(path)
+        default = os.path.join(settings.BASE_DIR, path)
+        parser.add_argument(
+            '-p',
+            '--path',
+            help='the path to the Excel spreadsheet to load',
+            default=default)
+        parser.add_argument(
+            '-a',
+            '--append',
+            help='Append to existing guide',
+            action='store_true',
+            default=False)
+
+    def handle(self, *args, **options):
```

Comment:
I struggle with this question, so am curious to hear your thoughts--do you think we need to unit test infrequently-run loader code like this?